tests/integration/component.go

```diff
@@ -236,6 +236,24 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
 		})
 
+		It("should describe not pushed component when it is created with json output", func() {
+			cmpDescribeJSON, err := helper.Unindented(helper.CmdShouldPass("odo", append(args, "create", "nodejs", "cmp-git", "--project", project, "--git", "https://github.com/openshift/nodejs-ex", "--context", context, "--app", "testing", "-o", "json")...))
+			Expect(err).Should(BeNil())
+			expected, err := helper.Unindented(`{"kind": "Component","apiVersion": "odo.openshift.io/v1alpha1","metadata": {"name": "cmp-git","namespace": "` + project + `","creationTimestamp": null},"spec":{"app": "testing","type":"nodejs","source": "https://github.com/openshift/nodejs-ex","ports": ["8080/TCP"]}, "status": {"state": "Not Pushed"}}`)
+			Expect(err).Should(BeNil())
+			Expect(cmpDescribeJSON).To(Equal(expected))
+			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
+		})
+
+		It("should describe pushed component when it is created with json output", func() {
+			cmpDescribeJSON, err := helper.Unindented(helper.CmdShouldPass("odo", append(args, "create", "nodejs", "cmp-git", "--project", project, "--git", "https://github.com/openshift/nodejs-ex", "--context", context, "--app", "testing", "-o", "json", "--now")...))
```

Comment:
Use local nodejs source to save few minutes of CI time

Reply:
fixed